va/caa.go

```diff
@@ -170,22 +172,19 @@ func (va *ValidationAuthorityImpl) getCAASet(ctx context.Context, hostname strin
 // unmodified *dns.CAA records that were processed/filtered are returned as the
 // third argument. Any  errors encountered are returned as the fourth return
 // value (or nil).
-func (va *ValidationAuthorityImpl) checkCAARecords(
-	ctx context.Context,
-	identifier core.AcmeIdentifier,
-	challengeType *string) (bool, bool, []*dns.CAA, error) {
-	hostname := strings.ToLower(identifier.Value)
+func (va *ValidationAuthorityImpl) checkCAARecords(ctx context.Context, ident core.AcmeIdentifier, params *caaParams) (bool, bool, []*dns.CAA, error) {
```

Comment:
Ditto re not collapsing this back to one line/changing the `identifier` arg name.

Reply:
Also unwound.